packages/app/src/app/pages/Pro/index.tsx

```diff
@@ -103,7 +125,7 @@ const Pro: React.FC = () => {
                           </Button>
 
                           <HelpText>
-                            You will be billed on the <b>30th</b> of each month.
+                            You will be billed on the <b>29th</b> of each month.
```

Comment:
Shouldn't this be `28th`, since February can also have only 28 days? 🤔

Reply:
copied pasted from `/patron` 🤷‍♀

Reply:
turns out they were completely wrong 😅. we charge folks the same day they got their subscription :P